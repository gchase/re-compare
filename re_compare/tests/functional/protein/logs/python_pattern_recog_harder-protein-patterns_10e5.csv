regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,7.421970367431641,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,7.421970367431641,"[-1, -1]"
DHKNLD.D,first_match,0,0.16307830810546875,"[-1, -1]"
DHKNLD.D,all_matches,0,0.16307830810546875,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,4.550933837890625,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,4.550933837890625,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.017881393432617188,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,4.568815231323242,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,0.21195411682128906,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,0.21195411682128906,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,6.516933441162109,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,6.516933441162109,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,2.2461414337158203,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,2.2461414337158203,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,3.2591819763183594,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,3.2591819763183594,"[-1, -1]"
P..L...KWE.C,first_match,0,0.17309188842773438,"[-1, -1]"
P..L...KWE.C,all_matches,0,0.17309188842773438,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,3.6699771881103516,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,3.6699771881103516,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,0.6000995635986328,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,0.6000995635986328,"[-1, -1]"
