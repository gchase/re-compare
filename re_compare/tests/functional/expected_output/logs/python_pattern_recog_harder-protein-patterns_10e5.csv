regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,3.9162635803222656,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,3.9162635803222656,"[-1, -1]"
DHKNLD.D,first_match,0,0.2357959747314453,"[-1, -1]"
DHKNLD.D,all_matches,0,0.2357959747314453,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,3.0028820037841797,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,3.0028820037841797,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.011920928955078125,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,3.014802932739258,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,0,0.17714500427246094,"[-1, -1]"
(GI)(G|R)P.Y..K..R,all_matches,0,0.17714500427246094,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,0,6.661891937255859,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,0,6.661891937255859,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,0,2.3708343505859375,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,0,2.3708343505859375,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,0,3.976106643676758,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,0,3.976106643676758,"[-1, -1]"
P..L...KWE.C,first_match,0,0.16498565673828125,"[-1, -1]"
P..L...KWE.C,all_matches,0,0.16498565673828125,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,2.6369094848632812,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,2.6369094848632812,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,0,0.9748935699462891,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,0,0.9748935699462891,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,2.0170211791992188,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,2.0170211791992188,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,5.72514533996582,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,0.17189979553222656,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,0.17189979553222656,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,0.007152557373046875,"[57, 58]"
(A|G....GK)(S|T),consecutive,1,0.007152557373046875,"[57, 58]"
(A|G....GK)(S|T),consecutive,2,0.00476837158203125,"[94, 95]"
(A|G....GK)(S|T),all_matches,2,0.011920928955078125,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,0,10.106086730957031,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,0,10.106086730957031,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,0,0.6382465362548828,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,0,0.6382465362548828,"[-1, -1]"
